### ISSUE TITLE
Make SpaceTx Format readable from FIJI

### ISSUE THREAD

**Issue by ambrosejcarr** [spacetx/starfish#1289]
3 users at the hackathon expressed an interest in being able to more easily examine data in SpaceTx Format. 

@joshmoore will the SpaceTx Format reader give us the capability to load our data into FIJI?

**Comment by joshmoore** [comment]
> will the SpaceTx Format reader give us the capability to load our data into FIJI?

Yes, exactly. (The presence of an OME-XML file _beside_ the `experiment.json` as produced by spacetx-writer is a workaround.)
